offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
I think it could be annoying to people to prefix their string values with "u" character.  Not everyone needs to know what unicode is and it is pretty obvious that this makes the configuration ugly.  One could ask why we do not prefix everything with "u" only some fields and the answer could be, because it makes our job easier sounds like a cheap excuse.  Yet this is my point of view and I encourage a discussion here.